src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -378,6 +378,7 @@ initialize_query_table()
   emplace_sysfs_get<query::flash_bar_offset>            ("flash", "bar_off");
   emplace_sysfs_get<query::is_mfg>                      ("", "mfg");
   emplace_sysfs_get<query::mfg_ver>                     ("", "mfg_ver");
+  emplace_sysfs_get<query::is_recovery>                 ("", "recovery");
```

Comment:
What is the difference between "mfg" and "recovery"?

Reply:
They are all shipped out with manufacturing state new card. MFG is used for old golden, recovery is used for newer card built by ssv2 tools.

Reply:
Thanks for the clarification.